docs/architecture/adr/0008-change-shared-modules-package-name.md

```diff
@@ -0,0 +1,51 @@
+# Change Shared Modules package to `net.thunderbird`
+
+- Issue: [#9012](https://github.com/thunderbird/thunderbird-android/issues/9012)
+
+## Status
+
+- **Proposed**
```

Comment:
In the index the status is accepted, in this file it is proposed. I think we can go ahead and approve this one.

I'll add a PR to suggest a mobile-planning post when there is a new ADR.